code/Test_definitions/connected-network-type-subscriptions.feature

```diff
@@ -0,0 +1,297 @@
+@Connected_network_type_subscription
+Feature: CAMARA Connected Network Type Subscriptions API, vwip - Operations on subscriptions
```

Comment:
Shouldn't we add here the release version?

Reply:
@akoshunyadi 
Usual practice is only to change version numbers from wip to the target version in the release PR itself (i.e. #244). So this PR should be merged with wip, and then #244 updated. Makes a bit more work, but means that there is only one PR that needs to be concerned about release version numbers.